lib/text.js

```diff
@@ -83,7 +65,25 @@ function sendText(phone, message, carrier, region, cb) {
   child.stdin.end();
 }
 
+//----------------------------------------------------------------
+/*  Overrides default config
+
+    Takes a new configuration object, which is
+    used to override the defaults
+
+    Params:
+      obj - object of config properties to be overriden
+*/
+function setConfig(obj) {
+  for (var prop in obj) {
```

Comment:
I think you could simplify this using `Object.assign` ([link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)):

```
config = Object.assign(config, obj);
```